File: manager-web/src/main/resources/static/js/service/brandService.js
//服务层
app.service('brandService',function($http){
	    	

	    //查找所有
        this.findAll = function () {
            return $http.get("./brand");
        }
    
        //分页查找
        this.findPage = function (pageNo, pageSize) {
            return $http.get("./brand/" + pageNo + "/" + pageSize);
        }
    
        //添加品牌数据
        this.save = function (entity) {
            return $http.post("./brand", entity);
        }
    
        //修改品牌数据
        this.update = function (entity) {
            return $http.put("./brand", entity);
        }
    
        //删除品牌数据
        this.delete = function (selectIds) {
            return $http.delete("./brand?ids=" + selectIds);
        }
    
        //模糊查询品牌数据
        this.search=function (pageNo, pageSize,entity) {
            return $http.post("./brand/search/" + pageNo + "/" + pageSize, entity);
        }
  	
});

File: sellergoods-interface/src/main/java/com/gm/service/BrandService.java
package com.gm.service;

import com.gm.common.PageResult;
import com.gm.common.Result;
import com.gm.pojo.TbBrand;

import java.util.List;

public interface BrandService {
    Result findAll();

    PageResult findPage(Integer pageNo, Integer pageSize);

    Result findPage(TbBrand brand, Integer pageNo, Integer pageSize);

    Result saveBrand(TbBrand brand);

    Result updateBrand(TbBrand brand);

    Result deleteBrand(Long[] ids);

    Result deleteOneBrand(Long id);
}
